package.json

```diff
@@ -12,11 +12,11 @@
   },
   "dependencies": {
     "@protobufjs/utf8": "^1.1.0",
-    "binaryen": "46.0.0-nightly.20180427",
+    "binaryen": "47.0.0-nightly.20180503",
     "glob": "^7.1.2",
     "long": "^4.0.0",
     "minimist": "^1.2.0",
-    "ts-node": "^5.0.1"
+    "ts-node": "^6.0.2"
```

Comment:
I had some issues with v6 when running the sources. Wasn't finding some things. Doesn't this happen for you?

Reply:
Hmm, In Mac OSX it seems everting ok with tests. How this checked?

Reply:
Iirc `npm run clean` `npm run test` `npm run build`, in this order, somehow triggered it.

Reply:
Are you sure it should be `I64` instead `i64`?

Reply:
Yeah, something like that. Ideally, we should fix the underlying issue, of course, so we can use v6 :)

Reply:
Ok, I fixed this =)

Reply:
Forgot about non-distribution requirements